src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Fix remove Azure File Directory fail with pipeline
```

Comment:
1. Please add the contents "Fixed https://github.com/Azure/azure-powershell/issues/11596".
1. We think `Fixed` is consistent with other logs.

Reply:
change log fixed